assembl/static2/js/app/components/harvesting/harvestingBox.jsx

```diff
@@ -0,0 +1,213 @@
+                  <div>
+                    <Translate value="harvesting.inProgress" />
+                  </div>
+                </div>
+              </div>
+            ) : (
+              <div className="validated-harvesting">
+                <span className="confirm-harvest-button assembl-icon-catch" />
+                <div className="harvesting-status-label">
+                  <div>
+                    <Translate value="harvesting.harvestSelection" />
+                  </div>
+                  <div>
+                    <Translate value="harvesting.validated" />
+                  </div>
+                </div>
+              </div>
+            )}
+          </div>
+          <div className="button-bar">
+            <Button disabled={disabled} className={classnames({ active: checkIsActive })}>
+              <span className="assembl-icon-check grey" />
+            </Button>
+            <Button disabled={disabled}>
+              <span className="assembl-icon-edit grey" />
+            </Button>
+            <Button disabled={disabled}>
+              <span className="assembl-icon-delete grey" />
+            </Button>
+            <Button disabled={disabled} className={classnames({ active: isNugget })}>
+              <span className="assembl-icon-pepite grey" />
+            </Button>
+            <span className="assembl-icon-ellipsis-vert grey" />
+          </div>
+          <div className="profile">
+            <AvatarImage userId={harvesterUserId} userName={harvesterUserName} />
+            <div className="harvesting-infos">
+              <div className="username">{harvesterUserName}</div>
+              {isExtract &&
+                extract &&
+                extract.creationDate && (
+                  <div className="harvesting-date" title={extract.creationDate}>
+                    {moment(extract.creationDate)
+                      .locale(contentLocale)
+                      .fromNow()}
+                  </div>
+                )}
+              {!isExtract && (
+                <div className="harvesting-date">
+                  <Translate value="harvesting.now" />
+                </div>
+              )}
+            </div>
+          </div>
+        </div>
+        <div className="harvesting-box-body">
+          {isExtract && extract && <div>{extract.body}</div>}
+          {!isExtract && <div>{selectionText}</div>}
+        </div>
+        {disabled && (
+          <div className="harvesting-box-footer">
+            <Button className="button-submit button-dark" onClick={this.validateHarvesting}>
+              <Translate value="common.attachFileForm.submit" />
+            </Button>
+            <Button className="button-cancel button-dark" onClick={cancelHarvesting}>
+              <Translate value="debate.confirmDeletionButtonCancel" />
+            </Button>
+          </div>
+        )}
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  contentLocale: state.i18n.locale
```

Comment:
can you get this from parent component ?

Reply:
No. The parent isn't connected.